src/components/views/rooms/NewRoomIntro.tsx

```diff
@@ -55,19 +55,21 @@ const NewRoomIntro = () => {
         ? room.targets[0]?.userId
         : DMRoomMap.shared().getUserIdForRoomId(roomId);
 
-    let body;
+    let body: JSX.Element;
     if (dmPartner) {
         let introMessage = _t("This is the beginning of your direct message history with <displayName/>.");
-        let caption;
+        let captionElement: JSX.Element = <></>;
```

Comment:
We don't ever initialise to empty fragments, shouldn't we stay self-consistent, e.g. with L58 for example

Reply:
If not initialising it, tsc in strict mode complains about using `captionElement` before assignment here https://github.com/matrix-org/matrix-react-sdk/pull/9231/files/e4c4b73b814288ba29a3f95605a6eb4830721d07#diff-c509de2dd1c9eb2add4073f662ec42078874b71b98d13e34d141abff5bb65f32L90 .

Reply:
null is valid for rendering, but you should be consistent with both vars either way